spark-on-suse/azuredeploy.json

```diff
@@ -0,0 +1,257 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUserName": {
+      "type": "SecureString",
+	  "metadata": { "description": "The Linux VM user name" }
+    },
+    "adminPassword": {
+      "type": "securestring",
+	  "metadata": { "description": "The LInux VM password" }
+    },
+    "sparkMasterSize": {
+      "defaultValue": "Basic_A1",
+      "type": "String",
+	  "metadata": { "description": "The size of master VM" }
+    }
+  },
+  "variables": {
+    "resourceGroupLocation": "[resourceGroup().location]",
+	"vmStorageAccountContainerName": "vhd",
+    "storageAccountName": "[concat(uniquestring(resourceGroup().id),'storageacc')]",
+    "virtualMachines_spark01_name": "sparkMaster",
+	"sparkMasterPublicIpName": "sparkMasterPublicIp",
+    "sparkNetWorkSecurityGroupName": "nsgroup",
+    "sparkVirtualNetworkName": "sparkVirtualNetwork",
+    "sparkMasterNetworkInterfaceName": "ni_master",
+    "networkInterfaces_slave1_name": "ni_slave1",
+    "sparkMasterInternalIP": "10.0.0.4",
+    "scriptUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/spark-on-suse/",
```

Comment:
See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#samples-that-contain-extra-artifacts-custom-scripts-nested-templates-etc for BP on including artifacts.